.scripts/compare-database.js

```diff
@@ -122,7 +122,23 @@ const queryDatabaseData = async (database) => {
     .map(async ({ table_name }) => {
       const { rows } = await pool.query(/* sql */`select * from ${table_name};`);
 
-      return [table_name, omitArray(rows, 'created_at', 'updated_at', 'secret', 'db_user', 'db_user_password')];
+      // check config rows except the value column
+      if (['logto_configs', '_logto_configs', 'systems'].includes(table_name)) {
+        return [table_name, omitArray(rows, 'value')];
+      }
+
+      return [table_name, omitArray(
+        rows,
+        'id',
+        'resource_id',
+        'role_id',
+        'scope_id',
+        'created_at',
+        'updated_at',
+        'secret',
+        'db_user',
+        'db_user_password'
+      )];
```

Comment:
Should we also omit `tenant_id`?

Reply:
i think no for OSS, since we'd like to restrict to two exact tenants: `default` and `admin`

Reply:
Then what if the user has solid knowledge about our multi-tenancy foundation and creates his own tenant? 🤔
Not to mention in our cloud preview, we are also going to use the same workflow for DB alterations, right?